tests/Transports.AspNetCore.Tests/AuthorizationHelperTests.cs

```diff
@@ -3,6 +3,8 @@
 
 namespace Tests;
 
+#pragma warning disable CS1998
```

Comment:
This should not be necessary.  Test.props has

`<WarningsNotAsErrors>$(WarningsNotAsErrors);CS1998</WarningsNotAsErrors>`

Perhaps we need to change something else, such as removing `-severity warn` from dotnet format.

Reply:
Ok with all other changes posted so far

Reply:
Or we should just switch it to `<NoWarn>$(NoWarn);CS1998</NoWarn>`.  The warning certainly does not affect tests.

Reply:
I moved it to nowarns.